### PR TITLE
Add article paths to storybook helper

### LINKED ISSUE #2740: Add article links to storybook helpers
**Is your feature request related to a problem? Please describe.**
Currently, we have titles for different articles in the various services listed down in `packages/utilities/psammead-storybook-helpers/src/text-variants`. 

**Describe the solution you'd like**
We should extend this list to include relevant links to those articles.

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by EinsteinNjoroge**
Resolves #2740 

**Overall change:** _Add article paths to storybook helper._

**Code changes:**

- _Add article paths to storybook helper._
- _Bump version and update documentation._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/utilities/psammead-storybook-helpers/src/text-variants.js [hunk 1]
```diff
@@ -27,69 +30,79 @@ const TEXT_EXAMPLES = {
...
     locale: 'my',
+    path: '/burmese/articles/c3w1kwwmm5yo',
   },
   news: {
     text: 'Could a computer ever create better art than a human?',
     longText:
       'The critic, author, poet and TV host was known for his witty commentary on international television.',
     script: 'latin',
     locale: 'en',
+    path: '/news/articles/cn7k01xp8kxo',
```
Comment (thekp) [inline packages/utilities/psammead-storybook-helpers/src/text-variants.js]: very nitpicky, but `news` is the only service that is not in alphabetical order. could you fix that pls 🙏

Comment (EinsteinNjoroge) [reply packages/utilities/psammead-storybook-helpers/src/text-variants.js]: 😄I like the attention to detail here, thanks @thekp

Comment (thekp) [reply packages/utilities/psammead-storybook-helpers/src/text-variants.js]: I've just been working on this package for a while now, just always forgot to update it myself 😅

**Review by thekp — APPROVED** [review]
Lets get this merged!

**Review by j-pendlebury — APPROVED** [review]
👍

**Pull request merged**